src/server/server_family.cc

```diff
@@ -64,11 +77,70 @@ ABSL_FLAG(bool, df_snapshot_format, true,
 ABSL_FLAG(int, epoll_file_threads, 0,
           "thread size for file workers when running in epoll mode, default is hardware concurrent "
           "threads");
+ABSL_FLAG(ReplicaOfFlag, replicaof, ReplicaOfFlag{},
+          "Specifies a host and port which point to a target master "
+          "to replicate. "
+          "Format should be <IPv4>:<PORT> or host:<PORT> or [<IPv6>]:<PORT>");
+ABSL_FLAG(bool, continue_on_replication_fail, false,
+          "When true, Dragonfly will continue operation "
+          "even if replication via '--replicaof' fails.");
 
 ABSL_DECLARE_FLAG(uint32_t, port);
 ABSL_DECLARE_FLAG(bool, cache_mode);
 ABSL_DECLARE_FLAG(uint32_t, hz);
 
+bool AbslParseFlag(std::string_view in, ReplicaOfFlag* flag, std::string* err) {
+#define BadFmt(m) \
+  do {            \
+    *err = m;     \
+    return false; \
+  } while (0)
+  if (in.empty()) {
+    flag->host = "";
+    flag->port = "";
+    return true;
```

Comment:
Shouldn't this be `false`? Did we actually parsed an empty flag?

Reply:
I _guess_ this is true in order to allow an empty flag value, i.e. this feature is not in use.